src/ape_test/_cli.py

```diff
@@ -12,9 +80,60 @@
     context_settings=dict(ignore_unknown_options=True),
 )
 @ape_cli_context()
+@click.option(
+    "-w",
+    "--watch",
+    is_flag=True,
+    default=False,
+    help="Watch for changes to project files and re-run the test suite with the given options.",
+)
+@click.option(
+    "--watch-paths",
+    multiple=True,
+    type=Path,
+    default=[Path("contracts"), Path("tests")],
```

Comment:
We will need to include compiling in the re-trigger and.or possibly put `.build/` here instead

Reply:
This works, my issue was unrelated to this.